drivers/include/mrf24j40.h

```diff
@@ -115,6 +115,10 @@ extern "C" {
 
 #define MRF24J40_MAX_FRAME_RETRIES      (3U)        /**< Number of frame retries (fixed) */
 
+#ifndef MRF24J40_USE_EXT_PA_LNA
+#define MRF24J40_USE_EXT_PA_LNA         (0U)        /**< Enable external PA/LNA control (Section 4.2 of datasheet) for MRF24J40MC/MD/ME */
```

Comment:
Thanks for addressing! Unfortunately this doesn't comply with out [Coding Conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions):"*Line length: aim for no more than 80 characters per line, the absolute maximum should be 100 characters per line*". I'd suggest you simply add ya `brief` like for example [here](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/at86rf2xx.h#L91)

Reply:
Ok. Sorry for this. I've done the changes.
Thanks !